apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -654,18 +679,20 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         vote_.executionScript = _executionScript;
         vote_.status = VoteStatus.Active;
 
-        // Notify the Agreement app tied to the current voting app about the vote created.
-        // This is mandatory to make the vote disputable, by storing a reference to it on the Agreement app.
+        // Notify the attached Agreement about the new vote; this is mandatory in making the vote disputable
+        // Note that we send `msg.sender` as the action's submitter--the attached Agreement may expect to be able to pull funds from this account
         vote_.actionId = _registerDisputableAction(voteId, _context, msg.sender);
 
         emit StartVote(voteId, msg.sender, _context);
     }
 
     /**
-    * @dev Internal function for a representative to cast a vote on behalf of many voters. It assumes the queried vote exists.
+    * @dev Cast votes on behalf of delegated voters as a representative.
     */
     function _voteOnBehalfOf(uint256 _voteId, bool _supports, address[] _voters) internal {
         Vote storage vote_ = _getVote(_voteId);
+        // Note that the period for representatives to vote can never go into a quiet ending
+        // extension, and so we don't need to check other timing-based pre-conditions
```

Comment:
Similarly, this one as well—voting as a representative has less "time-based" checks than when voting normally and I believe this explains why